infra-templates/nfs/0/docker-compose.yml

```diff
@@ -1,20 +1,27 @@
-nfs-driver:
-  privileged: true
-  image: rancher/storage-nfs:v0.1.1
-  pid: host
-  labels:
-    io.rancher.scheduler.global: 'true'
-    io.rancher.container.create_agent: 'true'
-    io.rancher.container.agent.role: environment
-  environment:
-    NFS_SERVER: '${NFS_SERVER}'
-    MOUNT_DIR: '${MOUNT_DIR}'
-    MOUNT_OPTS: '${MOUNT_OPTS}'
-  volumes:
-  - /run:/run
-  - /dev:/host/dev
-  - /var/lib/rancher/volumes:/var/lib/rancher/volumes:shared
-  log_driver: json-file
-  log_opts:
-    max-size: 25m
-    max-file: 2
+version: '2'
+services:
+  nfs-driver:
+    privileged: true
+    image: rancher/storage-nfs:v0.1.1
+    pid: host
+    labels:
+      io.rancher.scheduler.global: 'true'
+      io.rancher.container.create_agent: 'true'
+      io.rancher.container.agent.role: environment
+    environment:
+      NFS_SERVER: '${NFS_SERVER}'
+      MOUNT_DIR: '${MOUNT_DIR}'
+      MOUNT_OPTS: '${MOUNT_OPTS}'
+    volumes:
+    - /run:/run
+    - /dev:/host/dev
+    - /var/lib/rancher/volumes:/var/lib/rancher/volumes:shared
+    logging:
+      driver: json-file
+      options:
+        max-size: 25m
+        max-file: '2'
+    storage_driver:
```

Comment:
Am I missing something or was the `storage_driver` not in the previous `docker-compose.yml`? It might be better to add this in a separate PR.